packages-content-model/roosterjs-content-model-core/lib/publicApi/table/clearSelectedCells.ts

```diff
@@ -0,0 +1,28 @@
+import hasSelectionInBlock from '../selection/hasSelectionInBlock';
+import type { ContentModelTable, TableSelectionCoordinates } from 'roosterjs-content-model-types';
+
+/**
+ * Clear selection of a table.
+ * @param table The table model where the selection is to be cleared
+ * @param sel The selection coordinates to be cleared
+ */
+export function clearSelectedCells(table: ContentModelTable, sel: TableSelectionCoordinates) {
```

Comment:
Curious: can this be done by using `setSelection` and do not pass selectable parameter?

Reply:
Actually, I mean, it is possible to just use setSelection() function instead of this new function.

For example, consider you have a content model object:
```ts
const model: ContentModelDocument = { ... };
```

Now you want to clear any selection from this model, can you just do:
```ts
setSelection(model); // Passing in just the model parameter means we want to clear existing selections if any
```

This should clear all selections, including selection on the table. So we don't need to know the original selection at all.